src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidator.cs

```diff
@@ -134,7 +134,7 @@ public TimeSpan NonceLifetime
             {
                 if (value <= TimeSpan.Zero)
                 {
-                    throw LogHelper.LogException<ArgumentOutOfRangeException>(LogMessages.IDX10105, value);
+                    throw LogHelper.LogArgumentException<ArgumentOutOfRangeException>("value", LogMessages.IDX10105, value);
```

Comment:
I think argument name should be the name of property: NonceLifetime instead of "value". I think the message will read as:

System.ArgumentOutOfRangeException: IDX10603: IDX10105: NonceLifetime must be greater than zero. value: '<some value'"
Parameter name: NonceLifetime

We include the argument name in the message itself. So, the argument name in such cases will be redundant.